src/pages/studyView/table/tables.module.scss

```diff
@@ -9,7 +9,7 @@
 // The current portal is wrapped a class .cbioportal-frontend to overwrite the legacy styles
```

Comment:
you can get rid of this comment

Reply:
Don't we still have `.cbioportal-frontend`?